localisation/00_zuwc_united_localisation.csv

```diff
@@ -2888,6 +2881,36 @@ GetSocietyWe_we;we;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5035_kalimdor;Your ship has landed at the lifeless sandy shore of a new continent. The area around was dark and gloomy and you could hear your master voice more clear than ever. As soon as you stepped on the sand of southwestern Kalimdor, you felt that something was swarming under your feet. Soon you will journey west to break bonds of the Titans and free your Master.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5035_northrend;Your ship has landed at the lifeless frozen shore of a new island. The area around was dark and gloomy and you could hear your master voice more clear than ever. As soon as you stepped on this island, you felt that something was moving deep under your feet. Soon you will sail north to break bonds of the Titans and free your Master.;;;;;;;;;;;;;x
+EVTOPTA_WCDAC_5035;We should build a base here.;;Wir m�ssen weiter ziehen.;;;;;;;;;;;x
+EVTDESC_WCDAC_5040_kalimdor;You have led your followers to cavern complex deep under Tanaris and started creation of Twilight's Hammer base there. Caverns will protect you from both the sun and the prying eyes of the locals. Your pale orc followers are quite happy to live underground once again or perhaps it's the proximity to your Master that drove them into the state of ectasy.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5040_northrend;While exploring the island you have found a forgotten Titan complex. You have decided that the best course of action would be to defile it and build Twilight's Hammer base inside it. You have ordered your pale orc followers to start the construction which they began with vigorous frenzy no doubt inspired by their hate for Titans or perhaps it's the proximity to your Master.;;;;;;;;;;;;;x
+EVTOPTA_WCDAC_5040;Tendrils of Twilight's Hammer will spread from here.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5041_first_kill;Your guards have found one of your followers dead this morning. Upon closer inspection of the body you realized that it was not a result of brawl getting out of hand but an assasin's work. The cut was made with a precsision of a master killer but even more frightening was the fact that you have seen cuts like this many times during your time with Shadow Council.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5041_later_kills;This morning your guards have found another dead body inside your camp. The cut on its throat was the same as the one before and there could be no mistaking it for anything else than Garona's work. If the attacks continue the success of your mission could be compromised.;;;;;;;;;;;;;x
+EVTOPTA_WCDAC_5041;I know that blade... Garona.;;;;;;;;;;;;;x
+EVTOPTB_WCDAC_5041;I need to stop them.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_5042;After some time spend planning you have finally created a scheme that would allow you to capture Garona. Despite her mastery of assasination and subtlety there was one thing that made her vulnerable. She wanted above all things to hunt you down and kill you. Usin yourself as a living bait would allow your followers to corner and capture her.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCDAC_5042;After some time spend planning you have finally created a scheme that would allow you to capture Garona. Despite her mastery of assassination, there is one thing that makes her vulnerable. She has always greatly desired to hunt you down and kill you. Using yourself as a living bait would allow your followers to corner and capture her.;;;;;;;;;;;;;x
```